tools/torchfix/tests/fixtures/vision/checker/pretrained.py

```diff
@@ -0,0 +1,19 @@
+import torchvision
+import random
+
+# deprecated
+torchvision.models.resnet50(pretrained=True)
+torchvision.models.resnet50(True)
+torchvision.models.resnet50(pretrained=False)
+torchvision.models.resnet50(False)
+torchvision.models.detection.ssdlite320_mobilenet_v3_large(pretrained_backbone=True)
+torchvision.models.detection.ssdlite320_mobilenet_v3_large(pretrained=True, pretrained_backbone=True)
```

Comment:
Could we also add a test entry for [`resnet_fpn_backbone`](https://github.com/pytorch/vision/blob/main/torchvision/models/detection/backbone_utils.py#L68) and also one of the quantization models, e.g. [`resnet_18`](https://github.com/pytorch/vision/blob/7253af5827ad17f8ea40bcff12786d7b5c3cf3a8/torchvision/models/quantization/resnet.py#L292)?

For these we can't really suggest a `weight` replacement because it can only be determined at runtime, but maybe we can still show a LintViolation?


IIUC you already handled all of that properly, I just want to confirm through a test.

Reply:
Right now quantization models are not supported - nothing will happen.
I was thinking on dealing with that in a follow-up PR as this one is already large.

`resnet_fpn_backbone` also not included currently. Do you think it's OK to put this in the follow-up PR?

Reply:
Yes, that is fine.